templates/solace-vmr.template

```diff
@@ -1,60 +1,49 @@
 {
     "AWSTemplateFormatVersion": "2010-09-09",
-    "Description": "CloudFormation template to deploy Solace HA Message Routers on AWS. (qs-1nju7g5r0)",
+    "Description": "CloudFormation template to deploy Solace HA Message Routers on AWS",
```

Comment:
Keep the id: (qs-1nju7g5r0), it is used for tracking launches

Reply:
Fixed.